src/doc.d

```diff
@@ -1122,7 +1126,7 @@ extern (C++) void toDocBuffer(Dsymbol s, OutBuffer* buf, Scope* sc)
             // emit constraints if declaration is a templated declaration
             if (td && td.constraint)
             {
-                buf.writestring(" if (");
+                buf.writestring("$(DDOC_CONSTRAINT ");
```

Comment:
Where would that Phobos configuration be? I didn't see any ddoc config settings in Phobos -- is this something I have to create or am I looking in the wrong place?